tests/unit/utils/bind-data-test-attributes-test.js

```diff
@@ -129,6 +129,20 @@ test('it breaks if tagName is empty', function(assert) {
   assert.throws(() => bindDataTestAttributes(instance));
 });
 
+test('it does not breaks if tagName is empty and supportsDataTestProperties is set', function(assert) {
+  let Fixture = EmberObject.extend({
+    tagName: '',
+    supportsDataTestProperties: true,
+    'data-test-from-factory': 'foo',
+  });
+  let instance = Fixture.create({
+    'data-test-from-invocation': 'bar',
+  });
+
+  bindDataTestAttributes(instance);
+  assert.ok(true, 'did not throw');
```

Comment:
I would prefer `assert.expect(0);` at the top of the test instead of this constant-condition assertion

I wish QUnit had a `doesNotThrow()` assertion...